sdk/client/src/mod.rs

```diff
@@ -0,0 +1,130 @@
+// Copyright (c) The Diem Core Contributors
+// SPDX-License-Identifier: Apache-2.0
+
+use std::collections::HashMap;
+
+use crate::{Error, Result};
+use serde::{Deserialize, Serialize};
+
+mod blocking;
+mod client;
+mod request;
+mod response;
+
+pub use blocking::BlockingClient;
+pub use client::Client;
+pub use request::{JsonRpcRequest, MethodRequest};
+pub use response::{MethodResponse, Response, State};
+
+#[derive(Debug, Deserialize, Serialize)]
+enum JsonRpcVersion {
+    #[serde(rename = "2.0")]
+    V2,
+}
+
+#[derive(Debug, Deserialize, Serialize)]
```

Comment:
I'm struggling to understand the reason we need to redeclare all of these things in both lib.rs and mod.rs? Is there no way to get around this?

Reply:
oops, this file just shouldn't exist. I'll remove it. Thanks for catching it